awscli/clidocs.py

```diff
@@ -451,7 +453,13 @@ def _do_json_example(self, doc, argument_model, stack):
             doc.style.dedent()
             doc.write('}')
         elif argument_model.type_name == 'structure':
-            self._doc_input_structure_members(doc, argument_model, stack)
+            if argument_model.is_document_type:
+                doc.write("This value is an JSON document that can have arbitrary content.")
```

Comment:
s/an/a

If you're taking JSON here in the CLI, then you might say "JSON value" instead to clarify that any JSON value is accepted.